docs/Autotune.md

```diff
@@ -1,40 +1,41 @@
-# Autotune
+# Autotune (Rev. 2015-03-04)
 
 Autotune helps to automatically tune your multirotor.
 
 ## Configuration.
 
 Autotune only works in HORIZON or ANGLE mode, before using auto-tune it's best you setup so there is as little drift as possible.
-Autotuning is best on a full battery in good flying conditions, i.e. no or minimal wind. Autotune does not support 
-pid_controller 2 (pid_controller 0 is the Cleanflight default).
+Autotuning is best on a full battery in good flying conditions, i.e. no or minimal wind. Autotune does not support pid_controller 2  or higher (pid_controller 0 is the Cleanflight default, pid_controller 1 will work for autotune as well).
 
-Configure a two position switch on your transmitter to activate the AUTOTUNE and (HORIZON or ANGLE) modes using the auxiliary configuration.
-You may find a momentary switch more suitable than a toggle switch.
+Configure a two position switch on your transmitter to activate the AUTOTUNE mode. Autotune may be done in ether one of the both only, HORIZON or  ANGLE mode (will then apply on both modes). 
```

Comment:
meant "either".I do not have to do autotune twice - both in ANGELE - and again in HORIZON. Correct? So "either" may apply. --- Ursprüngliche Nachricht ---Von: Nicholas Sherlock <notifications@github.com>Datum: 04.03.2015 13:06:22An: cleanflight/cleanflight <cleanflight@noreply.github.com>Betreff: Re: [cleanflight] Update Autotune.md (#585)In docs/Autotune.md:
>
> -Configure a two position switch on your transmitter to activate the AUTOTUNE and (HORIZON or ANGLE) modes using the auxiliary configuration.
> -You may find a momentary switch more suitable than a toggle switch.
> +Configure a two position switch on your transmitter to activate the AUTOTUNE mode. Autotune may be done in ether one of the both only, HORIZON or  ANGLE mode (will then apply on both modes).

"ether one of the both only" isn't correct. The original text is better here.

—Reply to this email directly or view it on GitHub.